### PR TITLE
[Sprint 5] Load Data User When Login Success Add Create Activity Update Profile

### PR DESCRIPTION
**Pull request by Tuanlvt**
Task: Logic Profile
Link redmine: https://dev.framgia.com/redmine/issues/135500

### CODE REVIEW

File: app/src/main/java/com/framgia/fbook/screen/login/LoginActivity.kt [hunk 1]
```diff
@@ -82,6 +83,12 @@ open class LoginActivity : BaseActivity(), LoginContract.ViewModel {
   }
 
   override fun onUserLoggedIn() {
+    val abc: Int? = intent.getIntExtra(Constant.EXTRA_REQUEST_KEY, Constant.KEY_VALUE_DEFAULT)
```
Comment (HyperionsThs) [inline app/src/main/java/com/framgia/fbook/screen/login/LoginActivity.kt]: abc ???

File: app/src/main/java/com/framgia/fbook/screen/menuprofile/MenuProfileFragment.kt [hunk 2]
```diff
@@ -118,7 +125,15 @@ class MenuProfileFragment : BaseFragment(), MenuProfileContract.ViewModel {
   }
 
   fun onClickLogout() {
-    //Todo navigation Logout
+    val bundle = Bundle()
+    bundle.putInt(Constant.EXTRA_REQUEST_KEY, Constant.KEY_VALUE_ONE)
+    mUserRepository.clearData()
+    mNavigator.finishActivity()
+    mNavigator.startActivity(LoginActivity::class.java, bundle)
```
Comment (daolq3012) [inline app/src/main/java/com/framgia/fbook/screen/menuprofile/MenuProfileFragment.kt]: pls remove handle in here by show dialog confirm log out, 
if user choose ok, we only clear data and update layout MainActivity, 
Don't redirect to LoginActivity, It bad UX

File: app/src/main/res/values-vi/strings.xml [hunk 3]
```diff
@@ -31,4 +31,15 @@
     <string name="can_find_data">Không tìm thấy dữ liệu</string>
     <string name="you_must_be_login_into_perform_this_function">Bạn phải đăng nhập để thực hiện chức năng này. Vui lòng hãy đăng nhập !</string>
     <string name="inform">Thông báo</string>
+    <string name="are_u_want_to_sign_out">Bạn có chắc chắn muốn đăng xuất không?</string>
+
+    <!-- Dialog -->
+    <string name="agree">Đồng ý</string>
```
Comment (trainer-android-fr) [inline app/src/main/res/values-vi/strings.xml]: WARNING: [UnusedResources] The resource `R.string.agree` appears to be unused
 Unused resources make applications larger and slow down builds.

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -37,7 +37,7 @@
 
     <!-- Dialog -->
     <string name="agree">Agree</string>
-    <string name="disagree">Disagree</string>
+    <string name="cancel">Cancel</string>
```
Comment (trainer-android-fr) [inline app/src/main/res/values/strings.xml]: WARNING: [UnusedResources] The resource `R.string.cancel` appears to be unused
 Unused resources make applications larger and slow down builds.

File: app/src/main/res/values-vi/strings.xml [hunk 5]
```diff
@@ -31,4 +31,15 @@
...
+    <string name="cancel">Huỷ </string>
```
Comment (trainer-android-fr) [inline app/src/main/res/values-vi/strings.xml]: WARNING: [UnusedResources] The resource `R.string.cancel` appears to be unused
 Unused resources make applications larger and slow down builds.

File: app/src/main/res/values-vi/strings.xml [hunk 6]
```diff
@@ -31,4 +31,15 @@
...
+    <string name="allow">Cho phép</string>
```
Comment (trainer-android-fr) [inline app/src/main/res/values-vi/strings.xml]: WARNING: [UnusedResources] The resource `R.string.allow` appears to be unused
 Unused resources make applications larger and slow down builds.

File: app/src/main/res/values-vi/strings.xml [hunk 7]
```diff
@@ -31,4 +31,15 @@
...
+    <string name="deny">Từ chối</string>
```
Comment (trainer-android-fr) [inline app/src/main/res/values-vi/strings.xml]: WARNING: [UnusedResources] The resource `R.string.deny` appears to be unused
 Unused resources make applications larger and slow down builds.

File: app/src/main/res/values-vi/strings.xml [hunk 8]
```diff
@@ -31,4 +31,15 @@
...
+    <string name="dont_ask_again">Không hỏi lại</string>
```
Comment (trainer-android-fr) [inline app/src/main/res/values-vi/strings.xml]: WARNING: [UnusedResources] The resource `R.string.dont_ask_again` appears to be unused
 Unused resources make applications larger and slow down builds.

File: app/src/main/res/values-vi/strings.xml [hunk 9]
```diff
@@ -31,4 +31,15 @@
...
+    <string name="more_info">Xem thêm thông tin</string>
```
Comment (trainer-android-fr) [inline app/src/main/res/values-vi/strings.xml]: WARNING: [UnusedResources] The resource `R.string.more_info` appears to be unused
 Unused resources make applications larger and slow down builds.

File: app/src/main/res/values-vi/strings.xml [hunk 10]
```diff
@@ -31,4 +31,15 @@
...
+    <string name="choose">Chọn</string>
```
Comment (trainer-android-fr) [inline app/src/main/res/values-vi/strings.xml]: WARNING: [UnusedResources] The resource `R.string.choose` appears to be unused
 Unused resources make applications larger and slow down builds.

File: app/src/main/res/values-vi/strings.xml [hunk 11]
```diff
@@ -31,4 +31,15 @@
...
+    <string name="clear">Xoá</string>
```
Comment (trainer-android-fr) [inline app/src/main/res/values-vi/strings.xml]: WARNING: [UnusedResources] The resource `R.string.clear` appears to be unused
 Unused resources make applications larger and slow down builds.

File: app/src/main/res/values/strings.xml [hunk 12]
```diff
@@ -62,4 +62,5 @@
     <string name="following">Following</string>
     <string name="select_image_from_gallery_or_camera">Select image from gallery or camera</string>
     <string name="select_office">Select office</string>
+    <string name="are_u_want_to_sign_out">Are you sure want to sign out?</string>
```
Comment (trainer-android-fr) [inline app/src/main/res/values/strings.xml]: WARNING: [UnusedResources] The resource `R.string.are_u_want_to_sign_out` appears to be unused
 Unused resources make applications larger and slow down builds.

File: app/src/main/res/values-vi/strings.xml [hunk 13]
```diff
@@ -31,4 +31,15 @@
     <string name="can_find_data">Không tìm thấy dữ liệu</string>
     <string name="you_must_be_login_into_perform_this_function">Bạn phải đăng nhập để thực hiện chức năng này. Vui lòng hãy đăng nhập !</string>
     <string name="inform">Thông báo</string>
+    <string name="are_u_want_to_sign_out">Bạn có chắc chắn muốn đăng xuất không?</string>
```
Comment (trainer-android-fr) [inline app/src/main/res/values-vi/strings.xml]: WARNING: [UnusedResources] The resource `R.string.are_u_want_to_sign_out` appears to be unused
 Unused resources make applications larger and slow down builds.

**Pull request merged**